src/elements/element.arc.js

```diff
@@ -89,14 +88,17 @@ module.exports = Element.extend({
 		var pixelMargin = (vm.borderAlign === 'inner') ? 0.33 : 0;
 		var angleMargin;
 
+		var defaultColor = defaults.global.defaultColor;
```

Comment:
We've started placing aliased methods at the top of the file

Reply:
The challenge with that is if we capture it, someone creates a chart, and then changes the default color we'll never get the new value